js-wether/wether.js

```diff
@@ -0,0 +1,96 @@
+document.addEventListener("DOMContentLoaded", function () {
+    //指定されたプルダウンの値を取得
+    let getSelectValue = function (name) {
+        let result = "";
+        let elems = document.getElementById(name).options;
+        for (let i = 0, len = elems.length; i < len; i++) {
+            let elem = elems.item(i);
+            if (elem.selected) {
+                result = elem.value;
+                break;
+            }
+        }
+        return result;
+    };
+
+    //APIへの問い合わせ～HTML反映まで
+    let getAPI = function () {
+        //APIへの問い合わせURLを生成
+        let url = "http://api.openweathermap.org/data/2.5/weather?q=" + getSelectValue("city") + "&APPID=4b5774e9f3d2a07b84f0f2f88e486224";
+        //天気情報を取得＆HTMLに反映
+        let xhr = new XMLHttpRequest();
+        xhr.open("GET", url, true);
+        xhr.onload = function () {
+            let data_json = this.response;
```

Comment:
（要修正）
キャメルケースとスネークケースが入り混じっているので統一してください。
javascriptの場合は、キャメルケースを利用するほうが一般的

※その他にも同じ修正があるので対応をお願いします

```suggestion
            let dataJson = this.response;
```